app/multitenant/dynamo_collector.go

```diff
@@ -451,6 +573,22 @@ func (c *dynamoDBCollector) Add(ctx context.Context, rep report.Report) error {
 		return err
 	}
 
+	// fourth, put it in memcache
+	if c.memcache != nil {
```

Comment:
Can we do this before we put into dynamodb pls? In fact, can we go in this order: memcache, dynamodb, s3?  That will allow reports to appear earlier in the UI.  OTOH the order needs to be s3, dyanamo when there is no memcache...